src/Datadog.Trace/CorrelationIdentifier.cs

```diff
@@ -7,6 +7,9 @@ namespace Datadog.Trace
     /// </summary>
     public static class CorrelationIdentifier
     {
+        internal static readonly string ServiceKey = "dd.service";
+        internal static readonly string VersionKey = "dd.version";
```

Comment:
Could rename to `ServiceVersionKey` for consistency with the suggestion to rename the TracerSetting.

Reply:
Yep good idea. Implemented in b796b8b70a8cc19c9363023a3887534f78897805